core/src/main/scala/org/http4s/ServerSentEvent.scala

```diff
@@ -24,13 +25,15 @@ import org.http4s.util.{Renderable, Writer}
 import scala.util.Try
 
 final case class ServerSentEvent(
-    data: String,
+    data: Option[String] = None,
     eventType: Option[String] = None,
     id: Option[EventId] = None,
-    retry: Option[Long] = None
+    retry: Option[Long] = None,
```

Comment:
Do you want the `FiniteDuration` before 0.22 ABI gets locked?

Reply:
why yes I do thanks for remembering

Reply:
pushed